config/default.yml

```diff
@@ -1139,6 +1139,14 @@ Lint/DefEndAlignment:
     - def
   AutoCorrect: false
 
+Lint/InheritException:
+  # The default base class in favour of `Exception`.
+  EnforcedStyle: runtime_error
+  SupportedStyles:
+    - runtime_error
+    - standard_error
+  AutoCorrect: true
```

Comment:
You only need `AutoCorrect: false` to disable an existing auto-correct function. Otherwise it's enabled by default if `autocorrect` is implemented.